developments_build/05_export.sh

```diff
@@ -2,41 +2,42 @@
 source config.sh
 
 # Generate output tables
-psql $BUILD_ENGINE -v VERSION=$VERSION -v CAPTURE_DATE=$CAPTURE_DATE -f sql/export.sql
+psql $BUILD_ENGINE\
+    -v VERSION=$VERSION\ 
+    -v CAPTURE_DATE=$CAPTURE_DATE\
+    -f sql/export.sql
 
```

Comment:
I'm not sure much of the commented-out code is useful. I think it is worth starting over. Maybe just remove.

Reply:
sure